src/main/java/com/parasoft/demoapp/repository/industry/OrderRepository.java

```diff
@@ -15,6 +16,8 @@ public interface OrderRepository extends JpaRepository<OrderEntity, Long> {
 
     Page<OrderEntity> findAllByRequestedBy(String requestedBy, Pageable pageable);
 
+    Page<OrderEntity> findAllByStatusNotAndStatusNot(OrderStatus status1, OrderStatus status2, Pageable pageable);
```

Comment:
I'm not that familiar with the JpaRepository operations but I bet there should be something like "findAllByStatusNotIn" and those two status can be put into an array or a list, right? Please check and make changes if I'm right.
Because "findAllByStatusNotAndStatusNot" has a bad method name and limit the method to exclude exact two status.

Reply:
I have changed to use "findAllByStatusNotIn" after check, thanks!